dts/bindings/sensor/nxp,fxls8974-common.yaml

```diff
@@ -41,3 +41,14 @@ properties:
       - 8
       - 4
       - 2
+
+  expected-whoami-value:
+    type: int
+    default: 0x86
```

Comment:
default needs justification

although more likely this should be a required property

Reply:
Understood, I have updated the code with updates to all the boards with this binding.